test/remote/gateways/remote_datatrans_test.rb

```diff
@@ -183,6 +183,34 @@ def test_successful_void
     assert_equal response.authorization, nil
   end
 
+  def test_successful_store_purchase_unstore_flow
```

Comment:
💬 Comment:

Event when I really really like the confidence that this approach produces 👍 , having separted tests for store and unstore would be great to have more control on how and why a test could be failing.